api/en/336_Register_ComplexTypeProperty.md

```diff
@@ -86,8 +86,8 @@ JSON
 |Name|ComplexTypeProperty name|Number of digits: 1 - 128<br>Character type: Single-byte alphanumeric characters, hyphens ("-"), and underscores ("\_")<br>However, the string cannot start with a single-byte hyphen ("-") or underscore ("\_")|Yes||
 |_ComplexType.Name|ComplexType name attached|Number of digits: 1 - 128<br>Character type: Single-byte alphanumeric characters, hyphens ("-"), and underscores ("\_")<br>However, the string cannot start with a single-byte hyphen ("-") or underscore ("\_")|Yes||
 |Type|Type definition|Edm.Boolean / Edm.String / Edm.Int32 / Edm.Single / Edm.Double / Edm.DateTime / Registered ComplexType name|Yes||
-|Nullable|Null value authorization|true / false<br>The default value is Null|No||
-|DefaultValue|Default value|See the table below<br>The default value is Null|No||
+|Nullable|null value authorization|true / false<br>The default value is null|No||

```

Comment:
"Null value authorization" is English therefore, big N.

Reply:
My Instruction was wrong, so modify as above.